match2/wikis/starcraft2/legacy_player_cross_table.lua

```diff
@@ -57,7 +57,7 @@ function LegacyPlayerCrossTable.playerCrossTableToMatch2(frame)
 		end
 	end
 
-	newArgs.hide = 'true'
+	newArgs.hide = args.hide == 'false' and 'false' or 'true'
```

Comment:
If you say `and 'false'`, then the statement that is a part of will never be true, right? Or am I wrong?

Reply:
since `args.hide == 'false'` is boolean it will only take the `'false'` if that bool is true, else it will take `'true'`
could also write it as
```if args.hide == 'false' then
	newArgs.hide = 'false'
else
	newArgs.hide = 'true'
end

Reply:
Wouldn't this be the same then? Or can `args.hide` have different values than just `true` or `false`?

```suggestion
	newArgs.hide = args.hide or 'true'
```

Reply:
it is a param for manual input, so there might be utter bs in it

Reply:
Cool, looks good then!